DatadogTrace/Sources/DatadogTracer.swift

```diff
@@ -7,62 +7,29 @@
 import Foundation
 import DatadogInternal
 
-/// Datadog - specific span `tags` to be used with `tracer.startSpan(operationName:references:tags:startTime:)`
-/// and `span.setTag(key:value:)`.
-public enum DatadogSpanTag {
-    /// A Datadog-specific span tag, which sets the value appearing in the "RESOURCE" column
-    /// in traces explorer on [app.datadoghq.com](https://app.datadoghq.com/)
-    /// Can be used to customize the resource names grouped under the same operation name.
-    ///
-    /// Expects `String` value set for a tag.
-    public static let resource = "resource.name"
-    /// Internal tag. `Integer` value. Measures elapsed time at app's foreground state in nanoseconds.
-    /// (duration - foregroundDuration) gives you the elapsed time while the app wasn't active (probably at background)
-    internal static let foregroundDuration = "foreground_duration"
-    /// Internal tag. `Bool` value.
-    /// `true` if span was started or ended while the app was not active, `false` otherwise.
-    internal static let isBackground = "is_background"
-
-    /// Those keys used to encode information received from the user through `OpenTracingLogFields`, `OpenTracingTagKeys` or custom fields.
-    /// Supported by Datadog platform.
-    internal static let errorType    = "error.type"
-    internal static let errorMessage = "error.msg"
-    internal static let errorStack   = "error.stack"
-}
-
-/// Because `Tracer` is a common name widely used across different projects, the `Datadog.Tracer` may conflict when
-/// doing `import Datadog`. In such case, following `DDTracer` typealias can be used to avoid compiler ambiguity.
-///
-/// Usage:
-///
-///     import Datadog
-///
-///     // tracer reference
-///     var tracer: DDTracer!
-///
-///     // instantiate Datadog tracer
-///     tracer = DDTracer.initialize(...)
-///
-public class DatadogTracer: OTTracer {
+internal class DatadogTracer: OTTracer {
```

Comment:
Leaving it not renamed, because I feel we lack solid convention for such things. I can rename it to anything (`<whatever>Tracer`) but I wish we can make up some no-brainer pattern to follow - likely on next iOS Dev Chat.